### DISCUSSION TITLE
Welcome to wn-blocks-plugin Discussions!

### DISCUSSION THREAD

**Discussion by LukeTowers** [wintercms/wn-blocks-plugin#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by damsfx** [comment]
> To get started, comment below with an introduction of yourself and tell us about what you do with this community.

I'm Damien, a french dev for along 20 years, frontend or backend, nevermind, I love building things for both.   
I'm a big fan of WinterCMS, enjoying the WinterCMS spirit, his maintainers (who are so cool guys with really open mind) and his community members.  
I try to do my best to help the winter ecosystem with push requests, tests, french translations or ideas.
